docs/index.html

```diff
@@ -0,0 +1,133 @@
+<html>
+	<head>
+		<title>Grapher Documentation</title>
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+		<script type="text/javascript" src="https://vuejs.org/js/vue.js"></script>
```

Comment:
maybe use a specific version of vue ? So we don't head into other problems when this is updated in the future ?